deployment/services/cf-cdn.ts

```diff
@@ -15,10 +15,12 @@ export class CDNSecret extends ServiceSecret<{
 export function deployCFCDN({
   environment,
   s3,
+  s3Mirror,
   sentry,
 }: {
   environment: Environment;
   s3: S3;
+  s3Mirror: S3;
```

Comment:
in case someone is using Pulumi to self host they don't need to deploy mirror?

```suggestion
  s3Mirror: S3 | null;
```

Reply:
Yes but Pulumi is really for us, and only a reference for people. Making s3Mirror nullable makes no sense from perspective of our deployment.